integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.4.12
+0.4.13
```

Comment:
instead of changing this, please unlink the chip-build-android/version from chip-build/version and update only chip-build-android/version to 0.4.12.